### DISCUSSION TITLE
Turnkey LXC - It appears that no TurnKey LXC container was selected

### DISCUSSION THREAD

**Discussion by Topiatic** [tteck/Proxmox#3384] [Q&A] [answered]
I'm likely missing the obvious but being newer to this it's not yet obvious to me. I'm using the Turnkey LXC helper scripts (I've tried both with the same results) to try an install File Server on my Proxmox.

I get to the 'Select a TurnKey LXC to create:' part, use down arrow to highlight fileserver, hit enter and get "It appears that no TurnKey LXC container was selected". I've tried scrolling down to fileserver then tabing to Ok and hitting enter, same result. I've tried various key combos and just get the same result or the script just exits.

What am I missing or just don't know?

**Comment by tteck** [comment, marked as answer]
Use the spacebar to select

**Comment by Topiatic** [comment]
*sigh* 20 some years ago when I last played with linux and servers I knew that, just needed the reminder.

Thanks for the quick response and for your amazing scripts.
